tests/lora/test_lora_layers_flux.py

```diff
@@ -170,8 +167,7 @@ def test_lora_expansion_works_for_absent_keys(self):
         pipe.set_progress_bar_config(disable=None)
         _, _, inputs = self.get_dummy_inputs(with_generator=False)
 
-        output_no_lora = pipe(**inputs, generator=torch.manual_seed(0)).images
-        self.assertTrue(output_no_lora.shape == self.output_shape)
+        output_no_lora = self.cached_non_lora_outputs[self.scheduler_cls.__name__]
```

Comment:
Do we need to fetch based on scheduler class name now that we only use a single scheduler?